### PR TITLE
Added Devcontainer

### PR DESCRIPTION
**Pull request by RobinLinde**
Adding a devcontainer to help VS Code users on Windows to work on development of MapComplete.

Currently quite basic, but does seem to work quite nicely.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+	],
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [1234],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "npm install && npm run init",
```
Comment (pietervdvn) [inline .devcontainer/devcontainer.json:25]: Npm install is unneeded as npm run init does a npm ci

**Comment by pietervdvn** [comment]
Looks good, but how would the user flow then go? Is this detected automatically by VScode or should this be configured by the user?

In eithe case, there should be a mention of the dev container in 'Docs/Develop_deployment.md'

**Review by pietervdvn — CHANGES_REQUESTED** [review]
Please, update the documentation too

**Comment by RobinLinde** [comment]
> 
> 
> Looks good, but how would the user flow then go? Is this detected automatically by VScode or should this be configured by the user?
> 
> In eithe case, there should be a mention of the dev container in 'Docs/Develop_deployment.md'

I have added some documentation now (https://github.com/pietervdvn/MapComplete/pull/423/commits/8ddca45d242decde87421e2ca7032dc379f14006), but in short VS Code prompts the user after cloning the repo, on whether to use the devcontainer.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (RobinLinde) [reply .devcontainer/devcontainer.json]: Removed in https://github.com/pietervdvn/MapComplete/pull/423/commits/9ff02d2dba61ee15d465660dd94d02e04038d7da

**Review by pietervdvn — APPROVED** [review]

**Pull request merged**